src/script/api/script_goal.cpp

```diff
@@ -115,9 +115,10 @@
 	EnforcePrecondition(false, question != nullptr);
 	const char *text = question->GetEncodedText();
 	EnforcePreconditionEncodedText(false, text);
-	EnforcePrecondition(false, CountBits(buttons) >= 1 && CountBits(buttons) <= 3);
+	uint min_buttons = (type == ::GQT_QUESTION ? 1 : 0);
```

Comment:
New warning here. Probably shouldn't be globally namespaced? Or perhaps the parameter should be globally namespaced?